OpenRA.Mods.Cnc/UtilityCommands/ImportTiberianSunMapCommand.cs

```diff
@@ -221,7 +222,8 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 
 			// "btib" - Blue Tiberium
 			{
-				0x02, new byte[]
+				0x02,
+				new byte[]
```

Comment:
This is a bit awkward since it breaks consistency with e.g. Veins below. Does the style rule allow for `0x02, new byte[] {` (with the opening brace on the same line)?

Reply:
No, same line is also forbidden.